### PR TITLE
fixing listen snippet

### PR DESCRIPTION
**Pull request by GJJ2019**
Before :
Screenshot 2022-06-17 at 8 55 30 AM

After: 
Screenshot 2022-06-17 at 8 56 08 AM

### CODE REVIEW

**Comment by RobertBrunhage** [comment]
Thanks for the fix @GJJ2019!

Could you also update the `CHANGELOG.md` and `package.json` with the new version?

**Comment by GJJ2019** [comment]
done @RobertBrunhage

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,11 @@
 All notable changes to the **flutter-riverpod-snippets** extension will be documented in this file.
 
 Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
+### [1.1.1]
+
+### Fix
+
+- fixing `listen` snippet
```
Comment (RobertBrunhage) [inline CHANGELOG.md:10]:
```suggestion
- Fix `listen` snippet to use previous & next
```

**Review by RobertBrunhage — CHANGES_REQUESTED** [review]
Just a minor detail

**Review by RobertBrunhage — APPROVED** [review]
Thanks for the PR!

**Pull request merged**